autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>Branding: <code>Sagemaker</code> -&gt; <code>SageMaker </code> - do a case-sensitive search throughout the notebook.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>I could only find one that wasn't in code.  Fixed.  Thanks.</p>